jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ContainerConfigurationTemplate.java

```diff
@@ -118,6 +127,15 @@ public void setContainerCmd(List<String> cmd) {
     config.Cmd = cmd;
   }
 
+  public void setContainerExposedPorts(List<String> exposedPorts) {
```

Comment:
The template itself should probably be free of reproducibility logic. This can just be a TODO for now too.